common/lifestyle_perks/wc_physical_perks.txt

```diff
@@ -145,7 +145,7 @@ physical_tree_2_perk_2 = {
 	parent = physical_tree_2_perk_1
 }
 
-physical_tree_2_perk_3 = {
+stasis_trap = {
```

Comment:
The perk doesn't mention what it does.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/117035588-89a2d300-ad15-11eb-855c-d01590603795.png)

</details>